tests/test_oled.py

```diff
@@ -275,3 +275,27 @@ def test_display_multiline_text_with_newlines(oled, snapshot, fonts_mock):
     text = "Line1\nLine2\nLine3\nLine4"
     oled.display_multiline_text(text)
     snapshot.assert_match(to_bytes(oled.image), "defaults.png")
+
+
+def test_subscribes_to_pitopd_ready_event(oled):
+    from pitop.common.ptdm import Message
+
+    callback = oled._ptdm_subscribe_client._callback_funcs.get(Message.PUB_PITOPD_READY)
```

Comment:
it might be best to keep this as `__callback_funcs` and use the ._Class__callback_funcs trick to access it in the test, rather than change the actual code to make testing simpler